### PR TITLE
Updated to support mautic 2.0.0.

### PR DESCRIPTION
**Pull request by juanluisbaptiste**
Hi,

This PR updates the Dockerfiles to support the latest version of mautic 2.x, tested with docker 1.11.2.

https://www.mautic.org/latest.json still returns 1.2.4, I suppose is because the containers still haven't been updated ?

About the fpm image, I haven't used fpm before so I'm not sure how it is designed to be used. It requires an external web server like apache or nginx, or is the Dockerfile unfinished ?

### CODE REVIEW

**Comment by roderickm** [comment]
In the `php:5.6-apache` base image that I pulled, `/usr/src/php/php.ini-production` file wasn't available. That file was archived away in the `/usr/src/php.tar.xz` tarball. Tweaking the Dockerfile slightly fixed it up for me:

    RUN tar xf /usr/src/php.tar.xz \
            -C /usr/local/etc/php/ \
            --strip-components 1 \
            php-5.6.23/php.ini-production \
      && mv /usr/local/etc/php/php.ini-production /usr/local/etc/php/php.ini \
      && sed -i -e "/always_populate_raw_post_data/ s/^;//" /usr/local/etc/php/php.ini

**Comment by juanluisbaptiste** [comment]
Thanks, fixed it.

**Comment by roderickm** [comment]
Thanks, but I encountered that problem on the apache Dockerfile, pulling from the php:5.6-apache base image. I haven't tested the fpm deployment.

**Comment by juanluisbaptiste** [comment]
You are right, I dont know why it worked while I was working on this. Today I rebuilded the apache mautic image and I got the same proble. PR updated.

File: apache/Dockerfile [hunk 1]
```diff
@@ -28,6 +28,12 @@ RUN curl -o mautic.zip -SL https://s3.amazonaws.com/mautic/releases/${MAUTIC_VER
 COPY docker-entrypoint.sh /entrypoint.sh
 COPY makeconfig.php /makeconfig.php
 COPY makedb.php /makedb.php
+RUN tar xf /usr/src/php.tar.xz \
+        -C /usr/local/etc/php/ \
+        --strip-components 1 \
+        php-5.6.23/php.ini-production \
```
Comment (pierreozoux) [inline apache/Dockerfile]: This will break for each version of php, isn't it a better way of doing it? (It actually already break..)

**Comment by juanluisbaptiste** [comment]
Well, if you are talking about the php version, we could replace it with a build variable that can be passed with the correct version upon build time.

File: apache/Dockerfile (hunk 1 above)
Comment (juanluisbaptiste) [inline apache/Dockerfile]: Well, if you are talking about the php version, we could replace it with a build variable that can be passed with the correct version upon build time.

Comment (roderickm) [inline apache/Dockerfile]: Would it be more straightforward to include the php.ini to the mautic repo with `always_populate_raw_post_data` enabled and ADD it from the Dockerfile?

Alternatively, tar supports wildcards which would avoid the version-specific path, like so:
```
RUN tar xf /usr/src/php.tar.xz \
        -C /user/local/etc/php/ \
        --strip-components 1 \
        --wildcards php-*/php.ini-production \
```

Comment (juanluisbaptiste) [inline apache/Dockerfile]: I'm not familiar with `always_populate_raw_post_data`, so I'm not sure what you mean there. I can update the PR with the second suggestion tough.

**Comment by Gregy** [comment]
Hello,
thank you for the pull request. I am sorry I won't use it. I updated mautic to 2.7 and fixed the bugs mentioned here while adding cron support.

**Comment by juanluisbaptiste** [comment]
That's ok, the important thing is that we have an updated docker image !

**Pull request closed without merging**